pkgs/desktops/deepin/default.nix

```
@@ -0,0 +1,21 @@
+{ lib, pkgs, libsForQt5 }:
+let
+  packages = self: with self; {
```

Letś get rid of "`with`'s that  no one knows how it works":

```suggestion
  packages = self:
  let
    inherit (self) callPackage;
  in {
```